src/__tests__/swipe.test.js

```diff
@@ -0,0 +1,18 @@
+import { render } from "@testing-library/react";
+import ProfileCards from "../components/Swipe";
+import { useAuthMock } from "../mocks/setupTests";
+
+const auth = require("../services/AuthContext");
+describe("renderProfileCards", () => {
+  let mockAuth;
+  beforeEach(() => {
+    mockAuth = jest.spyOn(auth, "useAuth").mockImplementation(useAuthMock);
+  });
+  afterEach(() => {
+    mockAuth.mockRestore();
+  });
+
+  test("render profile: check if profile cards render", () => {
+    render(<ProfileCards />);
```

Comment:
rendering is good but I think we should add some things to expect() when it loads? I think if you could change the uid in src/mocks/setupTests.js then you could load the actual matches of someone. I know it's definitely not the best practice to have it query the database but that's probably the best we could do right now. Then you would expect to see a match card rendered. Just make sure the uid you change it to is one that has some matches in the database.